### PR TITLE
Fixes to support Quasar 2

### PR DESCRIPTION
**Pull request by jdfergason**
This merge request updates the component to work with Quasar 2:

- convert Vue.use and Vue.component to app.use and app.component in boot file
- fix rendering when there is no default slot, e.g. when src property is provided

### CODE REVIEW

File: ui/src/components/QMarkdown.js [hunk 1]
```diff
@@ -408,7 +408,7 @@ export default defineComponent({
 
         // get the markdown - slot overrides 'src'
         let markdown = source.value
-        if (slots.default()[ 0 ].children.trim().length > 0) {
+        if (slots.defaults !== undefined && slots.default()[ 0 ].children.trim().length > 0) {
```
Comment (hawkeye64) [inline ui/src/components/QMarkdown.js:411]: Looking good, but this should not be `slots.defaults` - just remove the `s` so I can merge. Thanks.

Comment (jdfergason) [reply ui/src/components/QMarkdown.js]: Good catch! I removed the s so you should be good to go.

**Comment by hawkeye64** [comment]
Thanks for the PR! :heart:

**Comment by hawkeye64** [comment]
Fix will be in v2.0.0-alpha.4 - releasing soon

**Pull request merged**